lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -507,6 +496,46 @@ def sectionNameFromSchema(self, schemaTypeName):
 
         return schemaTypeName
 
+    def createShaderAttributesSection(self):
+        """Use Sdr node information to populate the shader section"""
+        shader = UsdShade.Shader(self.prim)
+        nodeId = shader.GetIdAttr().Get()
+        if not nodeId:
```

Comment:
Should any form of error handling be done in these cases?

Reply:
Did you show this to Natalia?